### DISCUSSION TITLE
Plus installation issue

### DISCUSSION THREAD

**Discussion by GbekeAdesiyun** [PlusToolkit/PlusLib#999] [Q&A] [answered]
Hi,

There were no issues with the Plus installation on my computer ( the OS is Windows 11 for context ), but I keep getting error messages when I try to run it; I have attached the screenshots of the error messages. I downloaded the msvcp120.dll and msvcr120.dll files from the DLL-files.com website and pasted them in the Plus folder but that was futile. What can I do to fix this?
![image](https://user-images.githubusercontent.com/66129702/196768096-28c4e94b-23f3-4d2c-884f-6f825431fb6e.png)
![image](https://user-images.githubusercontent.com/66129702/196768143-fbdd1878-d0b2-4051-845c-3bb459d92c51.png)

Thanks,
Mogbekeloluwa Adesiyun

**Comment by adamrankin** [comment, marked as answer]
Have you installed the MSVC 2013 (and x64) redistributable?

**Reply by GbekeAdesiyun** [reply]
I have the MSVC 2015-2019 x64 and x86 redistributable. I have also repaired them. Do I need to install the 2013 version?

**Reply by adamrankin** [reply]
Yes, you need 2013.

**Reply by GbekeAdesiyun** [reply]
Ok, thanks
